staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -18,6 +18,11 @@ package v1beta1
 
 import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+// TODO: This does not generate to protobuf because of the interface{} - fix protobuf
+// generation to support a meta type that can accept any valid JSON.
+
+// +protobuf=false
```

Comment:
And probably an alias for `interface{}` like `type JSON interface{}` to write the custom marshaller for. @smarterclayton had you tried that for the table type?